### DISCUSSION TITLE
gfwlist和chinalist优先级问题

### DISCUSSION THREAD

**Discussion by niclau** [pymumu/smartdns#1763] [Q&A]
我设置了两个组分别是gfwlist和chinalist，
有一个域名，如：abcd.com在gfwlist里，而它的其中一个子域名是download.abcd.com在chinalist里，
我希望download.abcd.com不走透明代理（即直连），该如何实现？

**Comment by PikuZheng** [comment]
nameserver /download.abcd.com/-

或 

nameserver /-.abcd.com/gfwlist

或

调整配置文件顺序，chinalist后于gfwlist载入
